### PR TITLE
#154141622 Fix text component error 

### PR DESCRIPTION
**Pull request by TUgonna**
#### What does this PR do?
Fixes Text component error

#### Description of Task to be completed?
Fix the error that comes with the styling of the Text component 

#### How should this be manually tested?
- Check out into the `bg-text-component-error-154141622` branch.
- Run the app as well as tests to ensure everything is still in order.
#### Any background context you want to provide?
When trying to style the text component with `styled-components`, it throws an undefined error

#### What are the relevant pivotal tracker stories?
[#154141622](https://www.pivotaltracker.com/story/show/154141622)
#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: src/components/base/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ export { default as Box } from './Box';
 export { default as Button } from './Button';
 export { default as InputField } from './InputField';
 export { default as ShoppingCart } from './Cart';
-export { default as Modal } from './Modal';
 export { default as Text } from './Text';
 export { default as Cart } from './Cart';
 export { default as Loader } from './Loader';
+export { default as Modal } from './Modal';
```
Comment (fisayomi) [inline src/components/base/index.js]: Brilliant work @TUgonna

**Comment by oreoluwade** [comment]
@TUgonna I am curious about the number of commits you have in this branch, with some of them dating as far back as last year. Did you check out of develop to start this story?

**Comment by TUgonna** [comment]
@oreoluwade Yes I did checkout of develop

**Comment by omorilewa** [comment]
Looks good!

**Comment by omorilewa** [comment]
@TUgonna Did you see @oreoluwade request above?

**Comment by TUgonna** [comment]
@omorilewa Yup, done.
any other thing?
I need one approval to merge this.

**Comment by oreoluwade** [comment]
@TUgonna Hmmm. I am not sure one approval will be sufficient to merge it. Seems the coverage just dropped. Can you investigate why?

**Comment by oreoluwade** [comment]
@TUgonna Hmmm. I am not sure one approval will be sufficient to merge it. Seems the coverage just dropped. Can you investigate why?

**Comment by TUgonna** [comment]
I think The coverage decrease is from the current develop. @oreoluwade 
Which PR was merged last?

**Comment by TUgonna** [comment]
This is  running tests on develop
screen shot 2018-01-24 at 1 34 45 pm

**Comment by oreoluwade** [comment]
@TUgonna The tests on develop are passing (from the screenshot). That untested line 64 is expected. However, back on your branch, you'd notice that one particular test fails. Could you check what is causing it to fail?

**Comment by TUgonna** [comment]
@oreoluwade done

**Comment by oreoluwade** [comment]
@TUgonna One last thing. Could you please fix the indentation of the `Box component render` in the file you just made a change in?

**Comment by TUgonna** [comment]
@oreoluwade Which file? And what box component render?

**Comment by oreoluwade** [comment]
From lines 36 to 51 of the `base.spec.js` file. @TUgonna

**Comment by TUgonna** [comment]
Not sure what you mean but On my own end, the indentation is fine. And hound didn't alert me about it. @oreoluwade

**Comment by oreoluwade** [comment]
@TUgonna Hound does not flag indent errors (unlike flake8 in python).
Quick question: Is eslint enabled on your vscode? If it is, I suspect you may have seen some lines requiring a fix. Could you confirm this?

**Comment by TUgonna** [comment]
@oreoluwade Yes it is, still not seeing any lines in that file requiring a fix.
But this indentation looks fine to me though. am i missing something?
screen shot 2018-01-24 at 2 47 40 pm

**Comment by fisayomi** [comment]
@TUgonna , you might need to double-check that eslint is enabled. I can also see the indentation errors on my end. Also from the screenshot you sent, lines 39-50 should be indented backwards since we are using 2 line spaces.

**Comment by TUgonna** [comment]
screen shot 2018-01-24 at 2 53 51 pm

**Comment by TUgonna** [comment]
eslint actually shows errors when i try to indent backwards

**Pull request merged**